src/emulator/gxm/src/gxm.cpp

```diff
@@ -684,12 +729,28 @@ GLenum translate_minmag_filter(SceGxmTextureFilter src){
     }
 }
 
+GLenum translate_type(const SceGxmTextureFormat format)
+{
+    const auto base_format = texture::get_base_format(format);
+    switch (base_format) {
+        case SCE_GXM_TEXTURE_BASE_FORMAT_U4U4U4U4:
+            return GL_UNSIGNED_SHORT_4_4_4_4;
+        case SCE_GXM_TEXTURE_BASE_FORMAT_U8:
+        case SCE_GXM_TEXTURE_BASE_FORMAT_U8U8U8U8:
+            return GL_UNSIGNED_BYTE;
+        default:
+        {
+            return GL_UNSIGNED_BYTE;
```

Comment:
Should we warn about unhandled base formats?

Reply:
I removed these because we already warn _once_ in `sceGxmTextureInitLinear`, with the log here here it would spam the log every frame.

Reply:
Ah, that makes sense. Thanks!